.python-version

```diff
@@ -0,0 +1 @@
+3.11.6
```

Comment:
Personally I use pyenv for versions, and pyenv for virtualenv, so this file points to the name of my pyenv's virtualenv, e.g named `root` which is located in `.pyenv/versions/(venv)` and that's why cannot be deleted with the `make clean` command. I don't think this is inclusive enough approach.